roles/artemis/templates/artemis.env.j2

```diff
@@ -192,6 +192,9 @@ ARTEMIS_IRIS_SECRETTOKEN='{{ iris.secret }}'
 {% if enable_science_event_logging is defined %}
 ARTEMIS_SCIENCE_EVENTLOGGING_ENABLE='{{ enable_science_event_logging | lower }}'
 {% endif %}
+{% if artemis_theia_base_url is defined and artemis_theia_base_url is not none %}
+ARTEMIS_THEIA_BASE_URL='{{ artemis_theia_base_url }}'
```

Comment:
This env var does not match the key used in `roles/artemis/templates/application-prod.yml.j2` (`portal-url` vs `base.url`).
If you want to use `artemis.theia.portal-url` this would have to be like this:
```suggestion
ARTEMIS_THEIA_PORTALURL='{{ artemis_theia_base_url }}'
```

Reply:
Fixed